tests/time_test.cpp

```diff
@@ -37,9 +37,10 @@ BOOST_AUTO_TEST_CASE(time_point_sec_test)
     BOOST_CHECK_EQUAL( "20380119T031408", tp2g.to_non_delimited_iso_string() );
 
     time_point_sec tp3g(0xc0000000U);
-    // commented next 2 tests as they will only work with boost >= 1.64
-    //BOOST_CHECK_EQUAL( "2072-01-28T16:51:12", tp3g.to_iso_string() );
-    //BOOST_CHECK_EQUAL( "20720128T165112", tp3g.to_non_delimited_iso_string() );
```

Comment:
please remove this 3 commented lines. thanks.

Reply:
Already done. See raw file.

Reply:
ops, seems you are right, sorry.

Reply:
there aren't comments, removed already, look last commit please